pyface/timer/do_later.py

```diff
@@ -38,7 +38,7 @@ def do_later(callable, *args, **kwargs):
     ----------
     callable : callable
         The callable to call in 50ms time.
-    *args, **kwargs :
+    args, kwargs : tuple, dict
```

Comment:
Would this suggest `args` and `kwargs` can be passed as positional arguments?

Reply:
reverted the change to use `*args` and `**kwargs` again.